css/mmobile.css

```diff
@@ -109,6 +109,11 @@ h2 {
   background-image: none;
 }
 
+button {
```

Comment:
[OPTIONAL]

- [ ] Please, remember to always commit your work in the imperative tense. Here, you've used past tense for your commits. Kindly step back to review the lessons on making your repository professional.